clients/web/src/templates/widgets/metadataWidget.jade

```diff
@@ -1,11 +1,18 @@
-include metadatumViewMixin
 if (accessLevel >= girder.AccessType.WRITE)
-  button.btn.btn-sm.btn-primary.g-widget-metadata-add-button(title="Add Metadata")
-    i.icon-plus
+  .btn-group(style="display:block;float:right;")
+    button.g-widget-metadata-add-button.btn.btn-sm.btn-primary.btn-default.dropdown-toggle(data-toggle="dropdown", title="Add Metadata")
+      i.icon-plus
+    ul.dropdown-menu.pull-right(role="menu")
+      li(role="presentation")
+        a.g-add-simple-metadata(role="menuitem")
+          i.g-add-simple-metadata
+          | Simple
+      li(role="presentation")
+        a.g-add-json-metadata
+          i.g-add-json-metadata
```

Comment:
What is the `i` element here for (and the one just above this)? Did you mean to put some icons next to these?